src/Pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,19 @@
+import { CartList } from "../../Components/Cartlist/CartList";
+import { CartPrice } from "../../Components/CartPrice/CartPrice";
+import { useCart } from "../../Context/CartContext";
+import "./Cart.css";
+const Cart = () => {
+  const { cartState } = useCart();
+  return (
+    <div>
+      <h1>Cart ({cartState.cart.length})</h1>
+      <br />
+      <br />
```

Comment:
Can remove `<br/>` tags and use margins instead.

Reply:
Done..